android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -69,6 +69,11 @@ private Boolean isTablet() {
     return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
   }
 
+  @ReactMethod
+  public void getDeviceInfo(@NonNull final Promise promise) {
+    promise.resolve(this.getConstants().get("DeviceInfo"));
+  }
+
   @Override
   public @Nullable Map<String, Object> getConstants() {
```

Comment:
These are constants, I'm not quite following. 

We don't "change" these -- React Native calls this when it starts up, and what it returns it attaches to the JS Native Module.

We just also happen to have a "getter" for them